### DISCUSSION TITLE
Images width is small in landscape mode

### DISCUSSION THREAD

**Discussion by n-wissam** [gsantner/markor#1769] [General] [answered]
I noticed that  images width in landscape mode is small (30-40%) of the available space on screen. you can notice that more clearly on tablets
I added `img {width: 100%; height: auto;}` in settings to make it better
shouldn't that be the default behavior?

**Comment by gsantner** [comment, marked as answer]
Feel free to make a pull request with improvements that work fine for all cases. The place where the overall Markor CSS is, is here <https://github.com/gsantner/markor/blob/master/app/src/main/java/net/gsantner/markor/format/TextConverter.java#L47>. So is also the current applied `img { max-width: 100% }`.

**Reply by gsantner** [reply]
With the next Markor update new improvements are available.
